src/Stratis.Bitcoin.Features.WatchOnlyWallet.Tests/WatchOnlyWalletManagerTest.cs

```diff
@@ -181,10 +188,10 @@ public void Given_AWatchedAddress_ThenCanCalculateComplexRelativeBalance()
                 "0100000001dd19f894733ab950ae8e772ef781fe299f80f4e813ee4cb6ac067fedea3a052c000000006b483045022100d4d6570c054bcbafa8178e9074f2be32b078e0761537bfe4fb7ad1eea949ab66022022e31d6f43d9418b6dc02147aefb380a4730386364bd406365e790826400570b012102a31bf228b4508abe3f0aa5f91c53732dfd49233c2fef3c5cfa68f2aa12cc71b4ffffffff029e90bf00000000001976a914128240d302a4aadcdd08d241b54fa4ef11acb21388ace0c810000000000017a914188d767b139ef64ce4efa091c2957c6137fbe0ce8700000000"
             };
 
-            var block = new Block();
+            var block = this.network.Consensus.ConsensusFactory.CreateBlock();
```

Comment:
Any plans to move this like with the CreateTransaction method?

Reply:
For sure! We can do after this PR ;)